src/models/book.js

```diff
@@ -18,49 +18,177 @@ export default class extends Model {
     return type
   }
 
-  getImage(index) {
-    var data = this.get('images')[index]
-      , web = data.web
-      , type = this.getImageType()
+  decorateImage(image, index) {
+    var web = image.web
+      , {width, height} = _.pick(web, 'width', 'height')
 
     return {
-      ..._.pick(web, 'width', 'height')
-    , src: web.src[type]
+        index: index
+      , src: web.src[this.getImageType()]
+      , splited: false
+      , width: width
+      , height: height
+      , naturalWidth: width
+      , naturalHeight: height
+    }
+  }
+
+  @_.Memoize()
+  unsplitedImages() {
+    var images = this.get('images')
+    return images.map(::this.decorateImage)
+  }
+
+  @_.Memoize()
+  splitedImages() {
+    var images = this.get('images')
+
+    if (_.findIndex(images, (image) => { return image.width > image.height }) === -1) {
+      return this.unsplitedImages()
+    }
+
+    var maxWidthImage = _.max(images, (image) => { return image.width })
+      , beSplitedBreakpoint = maxWidthImage.width * 0.75
```

Comment:
https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints